### DISCUSSION TITLE
createSession(groupID, authorID, validUntil)

### DISCUSSION THREAD

**Discussion by fraengii** [ether/etherpad-lite#5061] [Q&A] [answered]
Hi,
what is the format of the value for 'validUntil'?
I've tryed '2022','2022-01-01','01.01.2022'. It always results: `{code: 1, message:"validUntil is in the past", data: null}`
My code:
```curl -G "http:/pad.domain/api/1.2.15/createSession?apikey=<APIKEY>&groupID=<groupID>&authorID=<authorID>&validUntil=2022"```

Regards
Fraengii

**Comment by webzwo0i** [comment, marked as answer]
https://github.com/ether/etherpad-lite/blob/develop/doc/api/http_api.md#createsessiongroupid-authorid-validuntil

an unix timestamp

**Reply by fraengii** [reply]
I'm so sorry!
I should take more time for really reading.
Thank you for answering anyway!
